lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -64,8 +64,11 @@ spec = do
             , input = "patate\n14\n"
             , expectedStdout =
                 "Prompt: ******\n\ESC[91mInt is an \
-                \integer number between -9223372036854775808 \
-                \and 9223372036854775807.\n\ESC[mPrompt: **\n"
+                \integer number between "
+                <> T.pack (show $ minBound @Int)
```

Comment:
ty <3

Reply:
although - I think @KtorZ fixed ascii colors as well in one of his PRs - so maybe the error msg has to be fixed as well